components/Overview.js

```diff
@@ -33,10 +55,12 @@ export default function Overview({ plants, handleToggleMyPlants }) {
       ) : (
         <ul>
           {plants.map((plant) => {
+            const isMyPlant = isMyPlantFunction(plant.id);
             return (
               <StyledList key={plant.id}>
                 <PlantPreview
                   plant={plant}
+                  isMyPlant={isMyPlant}
```

Comment:
Good solution. There is an option to make it less code, because the "const"-declarition isn't needed here: isMyPlant={isMyPlantFunction(plant.id)}